libnetwork/drivers/bridge/bridge_linux_test.go

```diff
@@ -582,9 +576,27 @@ func TestCreateFail(t *testing.T) {
 
 func TestCreateMultipleNetworks(t *testing.T) {
 	defer netnsutils.SetupTestOSContext(t)()
+	defer useStubFirewaller()()
 
 	d := newDriver(storeutils.NewTempStore(t))
 
+	checkFirewallerNetworks := func() {
+		t.Helper()
+		fw, ok := d.firewaller.(*firewaller.StubFirewaller)
+		assert.Assert(t, ok)
```

Comment:
What's the point of using a checked type-assertion if you're going to panic right after anyway?
```suggestion
		fw := d.firewaller.(*firewaller.StubFirewaller)
```